src/server/services/data-transform.service.test.js

```diff
@@ -475,91 +475,409 @@ describe("data-transform.service transformAnswersForSummary()", () => {
+              correctResponse.establishment_address_line_3
+            );
+          });
+        });
+
+        describe("establishment address line 3 is not null", () => {
+          const cumulativeAnswersEstAddSelected = {
+            establishment_address_selected: "0"
+          };
+
+          const testAddressLookupData = {
+            establishment_postcode_find: [
+              {
+                addressline1: "Example Street",
+                addressline2: "Example District",
+                addressline3: "Example Dependant Locality",
+                organisation: "The Organisation",
+                street: "Example Street"
+              }
+            ]
+          };
+          const correctResponse = {
+            establishment_address_line_1: "The Organisation, Example Street",
+            establishment_address_line_2: "Example District",
+            establishment_address_line_3: "Example Dependant Locality"
+          };
+
+          it("returns the response with the organisation included as address line 1", () => {
+            const response = transformAnswersForSummary(
+              cumulativeAnswersEstAddSelected,
+              testAddressLookupData
+            );
+            expect(response.establishment_address_line_1).toBe(
+              correctResponse.establishment_address_line_1
+            );
+            expect(response.establishment_address_line_2).toBe(
+              correctResponse.establishment_address_line_2
+            );
+            expect(response.establishment_address_line_3).toBe(
+              correctResponse.establishment_address_line_3
+            );
+          });
+        });
       });
-    });
-    describe("Given that only directly_export is part of cumulative answers", () => {
-      const cumulativeFullAnswers = {
-        directly_export: "True"
-      };
-      it("Should return a import_export_activities value of 'Directly export'", () => {
-        result = transformAnswersForSummary(cumulativeFullAnswers);
-        expect(result.import_export_activities).toBe("Directly export");
+
+      describe("operator address line 1 is equal to street but organisation is not present", () => {
+        const cumulativeAnswersEstAddSelected = {
+          operator_address_selected: "0"
+        };
+
+        const testAddressLookupData = {
+          operator_postcode_find: [
+            {
+              addressline1: "Example House",
+              addressline2: "Example Street",
+              addressline3: "Example Dependant Locality",
+              organisation: "",
+              street: "Example Street"
+            }
+          ]
+        };
+        const correctResponse = {
+          operator_address_line_1: "Example House",
+          operator_address_line_2: "Example Street",
+          operator_address_line_3: "Example Dependant Locality"
+        };
+
+        it("returns the response with the organisation included as address line 1", () => {
```

Comment:
the text description isn't accurate - "returns the response with the no changes to the address lines"